pkg/config/config.go

```diff
@@ -69,6 +69,16 @@ type LDAP struct {
 	UserFilter        string `mapstructure:"user_filter"`
 }
 
+// RemoteAuthenticator holds remote authentication configuration.
+type RemoteAuthenticator struct {
+	// BaseURL is the base URL of the remote authentication service (e.g. https://my-auth.com)
```

Comment:
example.com is one of a small number of TLDs that is [_guaranteed_ never to be assigned](https://www.rfc-editor.org/rfc/rfc2606.html) to anyone.  So I prefer always to use it in examples rather than point at some domain that might serve something my users won't like.  Also it's clearer that this is an example, and fewer people will think that they might be able to use it.

```suggestion
	// BaseURL is the base URL of the remote authentication service (e.g. https://my-auth.example.com)
```

Reply:
thanks, done